docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
   </Card>
-  <Card title="Customized to your needs" icon="rocket">
-    Terrateam offers a wide range of configuration options and extensibility points, allowing you to tailor the tool to your specific needs and workflows. You can define custom plan and apply steps, configure notifications and integrations, and leverage Terrateam's powerful features like drift detection, cost estimation, and policy enforcement.
+  
+  <Card title="Multi-Environment" icon="star">
+    Manage dev, staging, and production with consistent workflows.
+    <LinkCard title="Configuration" href="/configuration-reference" />
   </Card>
 </CardGrid>
 
-## Why Build When You Can Leverage?
-Building and maintaining a homegrown Terraform solution can be time-consuming, resource-intensive, and prone to errors. For a safe and effective workflow, you must account for the following:
-- Plan file storage and cleanup
-- Approval steps between Plan and Apply
-- Multiple pull requests with overlapping resources
-- Drift Detection and Reconciliation
-- Failed Apply operations
-- Locks and Unlocks
-- Plan and Apply restrictions
-- Secure cloud integrations
-
-Terrateam offers a battle-tested, feature-rich, and actively maintained solution that you can leverage out of the box. By adopting Terrateam, you can focus on writing Terraform code and managing your infrastructure, while benefiting from the collective experience and best practices of the Terrateam community.
-
 ## Getting Started
-To start using Terrateam, follow these simple steps:
 
 <Steps>
-1. #### Installation
+1. **Install**
 
-   Install the Terrateam GitHub application on your repository. See the [Quickstart Guide](/quickstart-guide) for detailed instructions.
+   Add the Terrateam GitHub app to your repository.
+   
+   <LinkCard title="Installation Guide" href="/quickstart-guide" />
 
-2. #### Configuration
+2. **Configure**
 
-   Configure Terrateam using the `.terrateam/config.yml` file in your repository. Check out the [Configuration Reference](/configuration-reference) for more information on the available options.
+   Create a `.terrateam/config.yml` file in your repository.
+   
+   <LinkCard title="Configuration" href="/getting-started/configuration" />
 
-3. #### Cloud Provider integration
+3. **Connect Cloud**
 
-   Set up authentication and authorization for your cloud provider. Terrateam supports various providers, including AWS, GCP, and Azure. See the [Cloud Providers](/cloud-providers) documentation for step-by-step instructions.
+   Set up secure OIDC authentication with your providers.
+   
+   <LinkCard title="Cloud Setup" href="/cloud-providers" />
 </Steps>
 
-## Learn More
-- Learn more about [Terrateam Concepts](/getting-started/concepts)
-- Dive into the [Configuration Reference](/configuration-reference) for a detailed look at the available options
-- Explore [Advanced Workflows](/advanced-workflows) and [Integrations](/integrations) to learn about complex use cases
+## Next Steps
```

Comment:
The "Next Steps" section could include a brief sentence to guide users according to their needs.